### ISSUE TITLE
/etc/systemd/logind.conf not actually supported by Systemd.lns

### ISSUE THREAD

**Issue by Artanicus** [hercules-team/augeas#602]
Attempted a simple modification:
```
%> sudo augtool
augtool> get /files/etc/systemd/logind.conf/Login/HandlePowerKey
/files/etc/systemd/logind.conf/Login/HandlePowerKey (o)
augtool> set /files/etc/systemd/logind.conf/Login/HandlePowerKey no
augtool> get /files/etc/systemd/logind.conf/Login/HandlePowerKey
/files/etc/systemd/logind.conf/Login/HandlePowerKey = no
augtool> save
error: Failed to execute command
saving failed (run 'errors' for details)
```

Relevant portion from errors:
```
Error in /etc/systemd/logind.conf at node /files/etc/systemd/logind.conf/files/etc/systemd/logind.conf/Login (put_failed)
  Failed to match tree under /files/etc/systemd/logind.conf/Login

     { "#comment" = "NAutoVTs=6" }
     { "#comment" = "ReserveVT=6" }
     { "#comment" = "KillUserProcesses=no" }
     { "#comment" = "KillOnlyUsers=" }
     { "#comment" = "KillExcludeUsers=root" }
     { "#comment" = "InhibitDelayMaxSec=5" }
     { "#comment" = "HandlePowerKey=poweroff" }
     { "#comment" = "HandleSuspendKey=suspend" }
     { "#comment" = "HandleHibernateKey=hibernate" }
     { "#comment" = "HandleLidSwitch=suspend" }
     { "#comment" = "HandleLidSwitchDocked=ignore" }
     { "#comment" = "PowerKeyIgnoreInhibited=no" }
     { "#comment" = "SuspendKeyIgnoreInhibited=no" }
     { "#comment" = "HibernateKeyIgnoreInhibited=no" }
     { "#comment" = "LidSwitchIgnoreInhibited=yes" }
     { "#comment" = "HoldoffTimeoutSec=30s" }
     { "#comment" = "IdleAction=ignore" }
     { "#comment" = "IdleActionSec=30min" }
     { "#comment" = "RuntimeDirectorySize=10%" }
     { "#comment" = "RemoveIPC=yes" }
     { "#comment" = "InhibitorsMax=8192" }
     { "#comment" = "SessionsMax=8192" }
     { "#comment" = "UserTasksMax=33%" }
     { "HandlePowerKey" = "no" }

  with pattern

(    { /Description/ }
      | { /Descriptio((n[.0-9A-Z_a-z-]|[.0-9A-Z_a-mo-z-])[.0-9A-Z_a-z-]*|)|Descripti([.0-9A-Z_a-np-z-][.0-9A-Z_a-z-]*|)|Descript([.0-9A-Z_a-hj-z
-][.0-9A-Z_a-z-]*|)|Descrip([.0-9A-Z_a-su-z-][.0-9A-Z_a-z-]*|)|Descri([.0-9A-Z_a-oq-z-][.0-9A-Z_a-z-]*|)|Descr([.0-9A-Z_a-hj-z-][.0-9A-Z_a-z-]*|
)|Desc([.0-9A-Z_a-qs-z-][.0-9A-Z_a-z-]*|)|Des([.0-9A-Z_abd-z-][.0-9A-Z_a-z-]*|)|De([.0-9A-Z_a-rt-z-][.0-9A-Z_a-z-]*|)|Environmen((t[.0-9A-Z_a-z-
]|[.0-9A-Z_a-su-z-])[.0-9A-Z_a-z-]*|)|Environme([.0-9A-Z_a-mo-z-][.0-9A-Z_a-z-]*|)|Environm([.0-9A-Z_a-df-z-][.0-9A-Z_a-z-]*|)|Environ([.0-9A-Z_
a-ln-z-][.0-9A-Z_a-z-]*|)|Enviro([.0-9A-Z_a-mo-z-][.0-9A-Z_a-z-]*|)|Envir([.0-9A-Z_a-np-z-][.0-9A-Z_a-z-]*|)|Envi([.0-9A-Z_a-qs-z-][.0-9A-Z_a-z-
]*|)|Env([.0-9A-Z_a-hj-z-][.0-9A-Z_a-z-]*|)|Exec([.0-9_-][.0-9A-Z_a-z-]*|[A-Za-z]|)|Exe([.0-9A-Z_abd-z-][.0-9A-Z_a-z-]*|)|(Ex[.0-9A-Z_a-df-z-]|E
n[.0-9A-Z_a-uw-z-]|(E[.0-9A-Z_a-mo-wyz-]|D[.0-9A-Z_a-df-z-]|[A-CF-Za-z][.0-9A-Z_a-z-])[.0-9A-Z_a-z-])[.0-9A-Z_a-z-]*|Ex|En|E[.0-9A-Z_a-mo-wyz-]|
D[.0-9A-Z_a-df-z-]|[A-CF-Za-z][.0-9A-Z_a-z-]/ }
      | { /Exec[A-Za-z][.0-9A-Z_a-z-]+/ }
      | { /Environment/ }
      | { /#comment/ = /[^\t\n\r ].*[^\t\n\r ]|[^\t\n\r ]/ }
      | { /\\.include/ = /[^\t\n #]*[^\t\n #\\]/ }
      | { })*

  Lens: /usr/share/augeas/lenses/dist/inifile.aug:459.27-460.17:
```

It's looking for a Description field of some kind, which is something the logind.conf will never have since it's not a systemd unit.

TL;DR; the systemd lens claims to support logind.conf but ignores the fact that it's a special cookie in systemd-land.

File contents as they come stock on Ubuntu 18.04:
```
#  This file is part of systemd.
#
#  systemd is free software; you can redistribute it and/or modify it
#  under the terms of the GNU Lesser General Public License as published by
#  the Free Software Foundation; either version 2.1 of the License, or
#  (at your option) any later version.
#
# Entries in this file show the compile time defaults.
# You can change settings by editing this file.
# Defaults can be restored by simply deleting this file.
#
# See logind.conf(5) for details.

[Login]
#NAutoVTs=6
#ReserveVT=6
#KillUserProcesses=no
#KillOnlyUsers=
#KillExcludeUsers=root
#InhibitDelayMaxSec=5
#HandlePowerKey=poweroff
#HandleSuspendKey=suspend
#HandleHibernateKey=hibernate
#HandleLidSwitch=suspend
#HandleLidSwitchDocked=ignore
#PowerKeyIgnoreInhibited=no
#SuspendKeyIgnoreInhibited=no
#HibernateKeyIgnoreInhibited=no
#LidSwitchIgnoreInhibited=yes
#HoldoffTimeoutSec=30s
#IdleAction=ignore
#IdleActionSec=30min
#RuntimeDirectorySize=10%
#RemoveIPC=yes
#InhibitorsMax=8192
#SessionsMax=8192
#UserTasksMax=33%
```

**Comment by raphink** [comment]
In `Systemd.lns`, `HandlePowerKey` will is parsed as an `entry_multi` type of field, with multiple `value` subnodes. You can see that by parsing the file with uncommented entries (for example using https://github.com/raphink/augeas-sandbox/blob/master/augcheck):

```
    { "HandlePowerKey"
      { "value" = "poweroff" }
    }
```


As a result, you need to set the `value`  subnode to set its value:

```
set Login/HandlePowerKey/value "no"
```
